swift-spaces-real-time-app/MuxSpacesSamples.xcodeproj/project.pbxproj

```diff
@@ -972,9 +982,10 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
-				CODE_SIGN_STYLE = Manual;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = XX95P4Y787;
```

Comment:
I'm curious why this was added. I thought we were removing a hard-coded development team for these examples.

Reply:
Good catch, removed this